src/components/home/ProjectsBrief.jsx

```diff
@@ -8,74 +8,105 @@ import CardContent from '@mui/material/CardContent';
 import CardMedia from '@mui/material/CardMedia';
 import Grid from '@mui/material/Grid';
 import Typography from '@mui/material/Typography';
+import { useMediaQuery } from '@mui/material';
 // Component Imports
 import projectsList from '../projects/projectsList';
+// Theme
+import { useTheme } from '@emotion/react';
 
-const cardStyle = {
-  display: 'flex',
-  justifyContent: 'space-between',
-  flexDirection: 'column',
-  m: { md: '0 25px 0 25px' },
-  p: { xs: 1, sm: 3 },
-  background: 'rgba(217, 217, 217, 0.4)',
-  borderRadius: '30px',
-  height: { xs: 'auto', sm: '100%' },
-  minHeight: { sm: '500px' }
+const cardStyle = (theme, isSmallScreen) => {
+  return {
+    display: 'flex',
+    justifyContent: 'space-between',
+    flexDirection: 'column',
+    m: { md: '0 25px 0 25px' },
+    p: { xs: 1, sm: 3 },
+    background: `${theme.palette.primary.cardFill}`,
+    borderRadius: '30px',
+    height: { xs: 'auto', sm: '100%' },
+    border:
+      theme.palette.mode === 'dark' ? `1px ${theme.palette.primary.boxOutline} solid` : 'none',
+    minHeight: { sm: '500px' },
+    backgroundImage:
+      theme.palette.mode === 'dark'
+        ? 'url(/assets/partnerLogos/backgroundBlobs/mobileBlob.webp)'
```

Comment:
The dark mode green is not the 'St John's Green' that matches the buttons. Desirable to be #56b6a1